packages/@jsii/dotnet-runtime-test/test/Amazon.JSII.Runtime.IntegrationTests/ComplianceTests.cs

```diff
@@ -1512,5 +1512,18 @@ public override String Repeat(String word)
                 return word;
             }
         }
+
+        [Fact(DisplayName = Prefix + nameof(ArrayOfInterfaces))]
```

Comment:
Do we need to implement this in all other languages?

What about arrays of enums?